src/ImageSharp/Formats/Gif/GifMetaDataExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.MetaData;
+
+namespace SixLabors.ImageSharp.Formats.Gif
+{
+    /// <summary>
+    /// Extension methods for storing meta data specific to Gif images.
+    /// </summary>
+    public static class GifMetaDataExtensions
+    {
+        /// <summary>
+        /// Adds or updates the Gif specific meta data to the image.
+        /// </summary>
+        /// <param name="meta">The image meta data.</param>
+        /// <param name="value">The gif meta data.</param>
+        public static void AddOrUpdateGifMetaData(this ImageMetaData meta, GifMetaData value) => meta.AddOrUpdateMetaData(GifConstants.MetaDataKey, value);
+
+        /// <summary>
+        /// Gets the Gif format specific meta data from the image.
+        /// </summary>
+        /// <param name="meta">The image meta data.</param>
+        /// <returns>The <see cref="GifMetaData"/> or null.</returns>
+        public static GifMetaData GetGifMetaData(this ImageMetaData meta)
+        {
+            meta.TryGetMetaData(GifConstants.MetaDataKey, out GifMetaData value);
+            return value;
```

Comment:
We should consider returning `new GifMetaData()`, when there is no value present for the key. This would simplify the call site a lot.

Reply:
I considered that actually. Probably wise.